### PR TITLE
Hide sensitive information from getting displayed/logged.

### LINKED ISSUE #535: Sensitive details are exposed in the build log
### Description of Problem, Request, or Question

When docker is used for building conan packages, entire docker command is included in the build log.
Therefore, sensitive information specified via CI environmental variables such as CONAN_LOGIN_USERNAME and CONAN_PASSWORD are also included in their pure form.
As this is a security issue, such sensitive information in the build logs has to be hidden or masked. 

### Environment Details
* Conan Package Tools Version: **0.32.1**
* Operating System: **Linux**
* Operation System Version: **Ubuntu 16.04**
* Compiler+version: **gcc-7**
* Docker image: **conanio/gcc7**
* Conan version: **conan 1.22.3**
* Python version: **python 3.7.0**

### Steps to reproduce (Include if Applicable)

No spcial steps needed.

### Build logs (Include if Available)

Azure pipelines build log file is attached.
CONAN_LOGIN_USERNAME and CONAN_PASSWORD has been masked.  
[log.txt](https://github.com/conan-io/conan-package-tools/files/5567941/log.txt)

### PR DESCRIPTION
**Pull request by kasunch**
Changelog: Fix: Fixes the issue with sensitive details are exposed in build log files

fixes #535 

- [x] Refer to the issue that supports this Pull Request.
- [x] If the issue has missing info, explain the purpose/use case/pain/need that covers this Pull Request.
- [x] I've read the [Contributing guide](https://github.com/conan-io/conan-package-tools/blob/develop/.github/CONTRIBUTING.md).
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008) style guides for Python code.

### CODE REVIEW

File: cpt/runner.py [hunk 1]
```diff
@@ -376,7 +377,11 @@ def __init__(self, runner, printer):
         self.printer = printer
 
     def __call__(self, command):
```
Comment (kasunch) [inline cpt/runner.py]: Opps..! Need to add "hide_sensitive=True" for the arguments.

**Review by kasunch — COMMENTED** [review]
Just copied the fix from an old source file and forgot to change the function arguments.

**Comment by uilianries** [comment]
It looks great. Is it possible add a test to check this new output?

**Comment by uilianries** [comment]
Depends on https://github.com/conan-io/conan/pull/8132

**Comment by uilianries** [comment]
Depends on https://github.com/conan-io/conan-package-tools/pull/538

**Pull request merged**